code/game/objects/machinery/doors/airlock_types.dm

```diff
@@ -261,6 +261,18 @@
 	icon = 'icons/obj/doors/mainship/securedoor.dmi'
 	req_access = list(ACCESS_MARINE_BRIDGE)
 
+/obj/machinery/door/airlock/mainship/ai
+	name = "\improper AI Core"
+	icon = 'icons/obj/doors/mainship/personaldoor.dmi'
+	req_access = list(ACCESS_MARINE_BRIG, ACCESS_MARINE_LOGISTICS)
```

Comment:
The previous one had req_one_access of 19, which is BRIDGE access, this one requires both brig and logistics.
if that's what you want to do it's fine

Reply:
Tbh idk what logistics is used for

Reply:
You should look it up where it's used (job datums have an access list), if you're not sure then you should preserve behavior, as this is changing it.

Reply:
ok dad